automobile/scenario-creation-tutorial.md

```diff
@@ -53,6 +53,11 @@ You can also enable the spline subdivision on some of the roads in Webots in ord
 
 ### Generate the SUMO network files
 
+> **Note**:
+A prerequisite on Linux is to run add the SUMO bin path in your LD_LIBRARY_PATH environment variable:
+
+        export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$SUMO_HOME/bin
```

Comment:
Since we do not define `SUMO_HOME` before I would rather use:
```
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$WEBOTS_HOME/projects/automobile/resources/sumo/bin
```